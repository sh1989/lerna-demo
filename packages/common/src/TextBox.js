import React from 'react';

const TextBox = ({ children }) => (
  <div className="tb">
    <h3>Look Some Text</h3>
    { children }
  </div>
);

export default TextBox;
