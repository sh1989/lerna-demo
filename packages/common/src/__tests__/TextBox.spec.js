import React from 'react';
import TextBox from '../TextBox';
import ShallowRenderer from 'react-test-renderer/shallow';

test('TextBox Component', () => {
    const renderer = new ShallowRenderer();
    renderer.render(
        <TextBox>
            <p>Text</p>
            <p>More Text</p>
        </TextBox>
    );
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
});