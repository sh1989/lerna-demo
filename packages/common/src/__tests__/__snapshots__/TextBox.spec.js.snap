// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextBox Component 1`] = `
<div
  className="tb"
>
  <p>
    Text
  </p>
  <p>
    More Text
  </p>
</div>
`;
